internal/services/apimanagement/testdata/api_management_api_openapi_invalide.yaml

```
@@ -0,0 +1,46 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+openapi: 3.0.0
+info:
+  title: api1
+  description: api
+  version: 1.0.0
+servers:
+  - url: "https://terraform.com/test/v1/api1"
+    description: test
+paths:
+  /default:
+    post:
+      operationId: default
+      summary: Default
+      description: Default operation
+      parameters:
+        - description: status
+          in: query
+          name: status
+          schema:
+            type: string
+            default: PRODUCTIVEInvalide
```

This looks like a typo? Should it be?
```suggestion
            default: PRODUCTIVEInvalid
```